network-api/networkapi/buyersguide/templates/buyersguide/catalog.html

```diff
@@ -139,6 +139,12 @@ <h1 class="h1-heading text-center mt-0">{{ page.header }}</h1>
               <div class="product-company body-small">{{product.company}}</div>
               <div class="product-name body">{{product.title}}</div>
             </a>
+            {% comment %}
+            FOR PR TESTING PURPOSES. THE NEXT THREE LINES OF CODE BELOW ARE TO BE REMOVED.
+            {% endcomment %}
+            {% for cat in product.product_categories.all %}
+            {{ cat }} <br>
+            {% endfor %}
```

Comment:
TO BE REMOVED

Reply:
it would be hilarious if we accidentally left this in =D